src/enum_net.cpp

```diff
@@ -377,6 +377,38 @@ namespace libtorrent
 		return false;
 	}
 
+#ifdef TORRENT_WINDOWS
+	HMODULE GetIPHelperHandle()
+	{
+		static bool handle_checked = false;
+		static HMODULE handle = 0;
+		if (!handle_checked)
```

Comment:
wouldn't it be sufficient to checkt ``if (handle)`` here? surely an ``HMODULE`` of 0 cannot be valid